tests/unit/s2n_certificate_test.c

```diff
@@ -804,5 +804,69 @@ int main(int argc, char **argv)
         EXPECT_BYTEARRAY_EQUAL(chain->sct_list.data, sct_list, sizeof(sct_list));
     };
 
+    /* Test s2n_cert_chain_and_key_load_cns */
+    {
+        DEFER_CLEANUP(struct s2n_cert_chain_and_key *chain = NULL,
+                s2n_cert_chain_and_key_ptr_free);
+        EXPECT_SUCCESS(s2n_test_cert_chain_and_key_new(&chain,
+                S2N_RSA_2048_SHA256_INTERMEDIATE_CERT_CUSTOM_OID,
+                S2N_RSA_2048_SHA256_INTERMEDIATE_CA_KEY));
```

Comment:
If you don't care, can you use the defaults? https://github.com/aws/s2n-tls/blob/main/tests/testlib/s2n_testlib.h#L174-L178 Using specific certs will make future devs think you need those specific certs for a reason.